src/Components/PlaylistModal/PlaylistModal.jsx

```diff
@@ -40,7 +41,7 @@ export const PlaylistModal = () => {
                                     value={playlist.title} 
                                     type="checkbox" 
                                     checked={isChecked} 
-                                    onChange={() => handlePlaylist(playlist._id, clickedPlaylistVideo, serviceListDispatch, isChecked)}
+                                    onChange={() => handlePlaylist(playlist._id, clickedPlaylistVideo, serviceListDispatch, isChecked, setToastData)}
```

Comment:
Line 16, instead of setNewPlaylistName(false); it should be setNewPlaylistName("");
This bug was showing on preview when I click on create new for 2nd new playlist.